MediaWikiKit/MediaWikiKit/MWKSavedPageList.m

```diff
@@ -35,14 +43,18 @@ - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     return self;
 }
 
-#pragma mark - Entry Access
-
-- (MWKSavedPageEntry*)entryAtIndex:(NSUInteger)index {
-    return [super entryAtIndex:index];
+- (void)importEntries:(NSArray*)entries {
+    NSArray<MWKSavedPageEntry*>* validEntries = [entries bk_reject:^BOOL (MWKSavedPageEntry* entry) {
```

Comment:
got lazy / wasn't sure if/where the best place was for validation logic of list entries.  e.g. we should only need to validate on import and rely on nullability annotations & assertions to catch the rest. but prior experience tells us we should probably assert & have fallbacks anyway. OK if i file a ticket to harden all `MWKList` objects against bad data?